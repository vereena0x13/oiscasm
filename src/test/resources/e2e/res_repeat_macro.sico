.macro resx(x) {
    .repeat 2 {
        .res 2, 1 << x
        .res 2, 2 << x
    }
}

resx(1)
resx(2)

.macro clear(x) {
    .repeat 2 x
    ?+1
}

clear(3)
clear(4)