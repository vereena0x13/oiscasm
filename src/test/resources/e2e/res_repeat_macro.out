2
2
4
4
2
2
4
4
4
4
8
8
4
4
8
8
3
3
19
4
4
22